.travis.yml

```diff
@@ -14,7 +14,8 @@ before_deploy:
   - git config --global user.email "builds@travis-ci.com"
   - git config --global user.name "Travis CI"
   - git remote set-url origin https://datree-ci:$GITHUB_TOKEN@github.com/datreeio/datree.git
-  - HOMEBREW_NO_AUTO_UPDATE=1 brew install caarlos0/tap/svu
+  - bash ./internal/release/install_svu.sh
+  - echo $(svu --version)
```

Comment:
add some context to the print here to make it easier to debug
`svu version to run` or whatever